src/Control/MonadZero.purs

```diff
@@ -32,13 +33,16 @@ import Data.Functor (class Functor, map, void, ($>), (<#>), (<$), (<$>))
 
 import Prim.TypeError (class Warn, Text)
 
+class Deprecated
```

Comment:
Will this work if we have multiple items deprecated? Perhaps we could use a pattern like ‘DeprecatedMonadZero’ if we need to distinguish classes in the future?

Reply:
I agree that making this class name more specific could help other issues in the future.

Reply:
I've renamed the class. Waiting on CI to build.

Reply:
I’m not sure to understand why we should rename the class actually. Don’t the core libraries encourage to qualify imports to disambiguate?

Reply:
I meant for within a single module when there are multiple deprecations; here it doesn’t really matter.

Reply:
Oh right, I didn’t pay attention to your first sentence initially for some reason 🤦

What do you think of an IsDeprecated suffix instead? I’m ok with whatever pattern you prefer though.

Reply:
That's fine as well -- just something more specific than `Deprecated`. So `MonadZeroIsDeprecated` is great, `DeprecatedMonadZero` is fine, whatever works!

Reply:
I like MonadZeroIsDeprecated because it's clearer than DeprecatedMonadZero. The former reads like a statement whereas the latter is left to interpretation.